### PR TITLE
Add BLOG_EMAIL to global context to make it available for templates

### PR DESCRIPTION
**Pull request by michaelb42**
### Pull Request Checklist

- [x] I’ve read the [guidelines for contributing](https://github.com/getnikola/nikola/blob/master/CONTRIBUTING.rst).
- [x] I updated AUTHORS.txt and CHANGES.txt (if the change is non-trivial) and documentation (if applicable).
- [x] I tested my changes.

### Description
Add BLOG_EMAIL to global context to make it available for templates. Fix Issue #2968

### CODE REVIEW

File: nikola/nikola.py [hunk 1]
```diff
@@ -451,6 +451,7 @@ def __init__(self, **config):
             'AUTHORLIST_MINIMUM_POSTS': 1,
             'BLOG_AUTHOR': 'Default Author',
             'BLOG_TITLE': 'Default Title',
+            'BLOG_EMAIL': 'Default EMail',
```
Comment (Kwpolska) [inline nikola/nikola.py]: Make it an empty string, please.

File: nikola/conf.py.in [hunk 2]
```diff
@@ -25,7 +25,7 @@ SITE_URL = ${SITE_URL}
 # This is the URL where Nikola's output will be deployed.
 # If not set, defaults to SITE_URL
 # BASE_URL = ${SITE_URL}
-BLOG_EMAIL = ${BLOG_EMAIL}
+BLOG_EMAIL = ${BLOG_EMAIL}  # (translatable)
```
Comment (Kwpolska) [inline nikola/conf.py.in]: I don’t think `BLOG_EMAIL` needs to be translatable. You can just make it a regular string (as `blog_url` is).

File: nikola/nikola.py [hunk 3]
```diff
@@ -656,6 +657,7 @@ def __init__(self, **config):
 
         self.TRANSLATABLE_SETTINGS = ('BLOG_AUTHOR',
                                       'BLOG_TITLE',
+                                      'BLOG_EMAIL',
```
Comment (Kwpolska) [inline nikola/nikola.py]: This one and `_GLOBAL_CONTEXT_TRANSLATABLE` (line 696) need to be removed as well.

Comment (michaelb42) [reply nikola/nikola.py]: oh, missed this one.

**Pull request merged**